core/partitioning/partitioning.cpp

```diff
@@ -40,10 +40,13 @@ bool containNonTensorOutputs(torch::jit::Node* n) {
 }
 
 bool isModifyingNodes(torch::jit::Node* node, torch::jit::Value* val) {
-  const auto& schema = node->schema();
+  const torch::jit::FunctionSchema* schema = node->maybeSchema();
```

Comment:
Do you think logging this check at the GRAPH level would be helpful for debugging?

Reply:
yes, I think so. What logging msgs would be good to have? @narendasan

Reply:
Probably something like `"<Whatever is doing the modifying> Is modifing node" << *node` or something